modules/islandora_oadoi/islandora_oadoi.install

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * @file
+ * Installation hooks for oaDOI badges.
+ */
+
+/**
+ * Implements hook_uninstall().
+ */
+function islandora_oadoi_uninstall() {
+  $vars = array(
+    islandora_oai_linktext,
```

Comment:
Ack, good catch... for some reason kept typing "oai" instead of "oadoi". Fixed with latest commit.

Not sure what you mean about the undefined constant though.